### PR TITLE
LPS-137224 Make export dialog not resizable

### PR DESCRIPTION
**Pull request by ramonmsp**

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/css/main.scss [hunk 1]
```diff
@@ -163,6 +163,7 @@ $multi-step-progress-bar-fixed-width: 100px !default;
 	.kaleo-process-preview-fields-dialog {
 		.modal-content {
 			.modal-body {
+				overflow-y: hidden;
```
Comment (naraandrade) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/css/main.scss:166]: @ramonmsp could you please break the line after this property?

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Review by jotascript — APPROVED** [review]

**Comment by ramonmsp** [comment]
ci:test:relevant

**Review by naraandrade — APPROVED** [review]

**Comment by ramonmsp** [comment]
ci:test:relevant

**Comment by ramonmsp** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
Hi @anabuchmann,

if this ci's run doesn't pass, please forward it manually to BChan.

Thank you

**Comment by anabuchmann** [comment]
Just started reviewing :)

**Comment by anabuchmann** [comment]
Manually forwarded here: https://github.com/brianchandotcom/liferay-portal/pull/105727

**Pull request closed without merging**